frontend/src/components/Name.svelte

```diff
@@ -0,0 +1,102 @@
+<script>
+  import { createEventDispatcher } from 'svelte/internal';
+
+  import Button from './Button';
+  import userState from '../state/user';
+
+  const messageCharacters = 'Only letters and numbers are allowed.';
+  const messageEmpty = 'Name cannot be empty.';
+
+  const dispatch = createEventDispatcher();
+
+  $: warning = checkName($userState.name);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    if (warning.length < 1) {
+      dispatch('submitted', true);
+    }
+  }
+
+  function checkName(name) {
+    if (!name || name.length < 1) {
+      return messageEmpty;
+    }
+
+    if (!/^[a-zA-Z0-9äöüß]+$/.test(name)) {
+      return messageCharacters;
```

Comment:
Poor André!

Reply:
I think only namenskürzel are supposed to be used here ;)